example/src/components/RoomParticipant.tsx

```diff
@@ -24,40 +35,75 @@ export const RoomParticipant = ({
     return false;
   };
 
+  const getTextForPinButton = () => {
+    return focused ? 'Unpin user' : titleSmall ? 'Pin' : 'Pin user';
+  };
+
+  const onPinButton = () => {
+    if (focused) {
+      onPinButtonPressed(null);
+      return;
+    }
+    onPinButtonPressed(videoTrack!.id);
```

Comment:
what if I click on a user without video track (it is possible and I can reproduce it by joining with camera off)? 
that's why using `!` is a bad practice :D

Reply:
Changed to use participant id instead. Videotrack id was way more convinient to use if you take screencast into consideration but there's a lot of edge cases that needed to be accounted for, so I think participant id will be the better choice.

Reply:
But as you said it will break for screencast, one user can have two video tracks: video and screencast.

Reply:
Yes, but I was planning to fix that with the screencast PR

Reply:
My new idea is to keep it with participantId and then in the screencast PR add additional parameter like `videoTrackType= 'camera' | 'screenshare'` and that's how I'will distinguish the two.

Reply:
sounds good